cmake/Modules/FindCapnp_EP.cmake

```diff
@@ -35,6 +35,20 @@ include(TileDBCommon)
 # If the EP was built, it will install the CapnProtoConfig.cmake file, which we
 # can use with find_package.
 
+if(WIN32)
```

Comment:
It seems like we should use the same version on all platforms? (disregard if already discussed elsewhere though)

Reply:
Yes, previous discussion chose current path of differing versions, I think, to reduce possible destabilization of existing production code when upgrading.
(Same version has been experimentally built for both *nix/win32.)

Reply:
This is fine by me -- it's inevitable that we will have mismatched capnp libraries among clients and servers. Since these protocol libraries guarantee backwards compatibility, I don't see an issue.